cmd/podman/stats.go

```diff
@@ -140,7 +140,7 @@ func statsCmd(c *cli.Context) error {
 				containerStats[id] = initialStats
 			}
 			stats, err := ctr.GetContainerStats(containerStats[id])
-			if err != nil {
+			if err != nil && errors.Cause(err) != libpod.ErrNoSuchCtr {
```

Comment:
If the err is libpod.ErrNoSuchCtr, should we instead continue and not do further processing in this loop?

Reply:
we do, we only stop if it isnt that.

Reply:
if the error is not ErrNoSuchCtr, we want to return here ... i think the logic is correct. am I missing something.

Reply:
I think @TomSweeneyRedHat has code that is a little more correct than this, but it shouldn't be a big difference (dropping the container one tick earlier). We can handle this in a followon. And hopefully clean the logic a bit more, because this is sort of difficult to read.